### DISCUSSION TITLE
Provide Billing Address for Subscriptions to Get A Proper Invoice

### DISCUSSION THREAD

**Discussion by medoingthings** [craftcms/commerce-stripe#169] [Ideas]
I fail to add the billing address to the Stripe customer, so that the address appears on the auto generated (subscription) invoice. I managed to add the address to the card information, but that address doesn’t appear on the invoice.

Ideally the address could be entered along with the card details on the payment method form.

Any hint is much appreciated!

**Comment by andris-sevcenko** [comment]
Currently no way of accomplishing this. Seems like this fits the bill for the long-term plans for this gateway - e.g. integrating deeper with Commerce concept of customer and syncing that with Stripe better.
